compose/ui/ui-graphics/samples/build.gradle

```diff
@@ -25,17 +25,17 @@ plugins {
 }
 
 dependencies {
-    kotlinPlugin(project(":compose:compiler:compiler"))
+    kotlinPlugin(projectOrArtifact(":compose:compiler:compiler"))
 
     implementation(libs.kotlinStdlib)
 
-    compileOnly(project(":annotation:annotation-sampled"))
+    compileOnly(projectOrArtifact(":annotation:annotation-sampled"))
 
-    api(project(":compose:ui:ui-unit"))
+    api(projectOrArtifact(":compose:ui:ui-unit"))
     implementation("androidx.compose.foundation:foundation:1.0.0")
-    implementation(project(":compose:runtime:runtime"))
-    implementation(project(":compose:ui:ui-graphics"))
-    implementation(project(":compose:ui:ui-util"))
+    implementation(projectOrArtifact(":compose:runtime:runtime"))
+    implementation(projectOrArtifact(":compose:ui:ui-graphics"))
```

Comment:
Why do these need to be projectOrArtifact? They are part of the playground.

Reply:
Done